docs/data/stashcache/run-stash-origin-container.md

```diff
@@ -43,14 +51,14 @@ set StashOriginPublicAuthfile = /etc/xrootd/public-origin-authfile
 ```
 
 
-### Disabling OSG monitoring ###
+### Disabling OSG Monitoring ###
```

Comment:
We don't use title case for h3's and higher: https://opensciencegrid.org/technology/documentation/style-guide/#headings

Reply:
And let's move the warning right under the heading

Reply:
```suggestion
### Disabling OSG monitoring (testing only) ###
```
perhaps?

Reply:
Actually, let's blow out this section entirely. This was added for CMS + ATLAS and I don't think there's really any cases where this would make sense for StashCache. I could be missing something, though. Thoughts @djw8605 @efajardo ? This is the config that it turns off: https://github.com/opensciencegrid/Software-Redhat/blob/trunk/osg-xrootd/osg/50-osg-monitoring.cfg#L12-L23

Reply:
So don't document how to turn off the transfer accounting?

Reply:
So, i updated the text to remove the use-case that (presumably) only applied to CMS + ATLAS, and only mentioned using this for testing, which we also say in the warning.  It seems like testing might still be a valid use-case to disable monitoring.

Reply:
> So don't document how to turn off the transfer accounting?

Yeah, is there a good reason that any StashCache hosts should disable this accounting?

Reply:
Not from my perspective, but I'm always pro-monitoring.

Reply:
@edquist burn this section to the ground, please

Reply:
CC @matyasselmeci in case we need to do something similar for the cache doc

Reply:
> edquist burn this section to the ground, please

you got it boss!